spec/poros/quote_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Quote do
+  it 'has attributes' do
+    attributes = {
```

Comment:
I would probably add in `content` which is the actual quote since it is one of the attributes you are wanting to display on the page.